minecraft_servers/griefergames/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+    "server_name": "GrieferGames",
+    "nice_name": "GrieferGames Netzwerk",
+    "direct_ip": "griefergames.net",
+    "social": {
+      "web": "https://griefergames.net/",
+      "web_shop": "https://store.griefergames.net/",
+      "web_support": "https://bit.ly/griefergames-support",
```

Comment:
Please use the direct url, not https://bit.ly/.

Reply:
URL is changed to the forum